SU-SWS/drupal-caravan
# SITES-126: Migrating using an inventory

# READY FOR REVIEW


# Summary
- Migrate sites to ACSF using an inventory

# Needed By (Date)
- This should be merged before we re-architect the migration tool

# Criticality
- How critical is this PR on a 1-10 scale? 4

# Steps to Test

1. Check out this branch
2. Run `ansible-playbook -i provisioning/inventory/sitestest migration-playbook.yml` and see what it does.
3. See `provisioning/inventory/sites` for an example of a larger inventory

# Affected Projects or Products
- Sites 2.0

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SITES-124
